src/languages/en-US/commands/tools.json

```diff
@@ -63,6 +63,17 @@
 		"OTHER": "Other",
 		"OVERVIEW": "Overview"
 	},
+	"createEmojiDescription": "Adds a custom emoji to your Discord server.",
+	"createEmojiExtended": {
+		"usages": ["Emoji"],
+		"extendedHelp": "Using a custom emoji on another server, I can re-upload that emoji to the current server, so non-nitro users can also access it.",
+		"explainedUsage": [["Emoji", "A custom emoji on Discord."]],
+		"reminder": "Make sure the bot has {{MANAGE_EMOJI, permissions}} permissions in this server!"
+	},
+	"createEmojiInvalidDiscordEmoji": "I could not resolve `{{parameter}}` to a valid Discord emoji. Make sure it is from a Discord server, and not a default emoji.",
+	"createEmojiDuplicate": "An emoji named `{{name}}` already exists!",
+	"createEmojiInvalidEmoji": "{{REDCROSS}} An invalid emoji was provided.",
```

Comment:
It could be valid in the sense that it was once created (and this will be, over 99% of the cases, people don't just forge incorrect emojis to pass the emoji format validation step).

Instead, you can try saying that you could not download the emoji, or something unexpected happened.